js/markdown-code/MarkdownCode.svelte

```diff
@@ -1,11 +1,12 @@
 <script lang="ts">
-	import { afterUpdate, tick } from "svelte";
+	import { afterUpdate, tick, onMount } from "svelte";
 	import render_math_in_element from "katex/contrib/auto-render";
 	import "katex/dist/katex.min.css";
 	import { create_marked } from "./utils";
 	import { sanitize } from "@gradio/sanitize";
 	import "./prism.css";
 	import { standardHtmlTags } from "./html-tags";
+	import mermaid from "mermaid";
```

Comment:
Mermaid can be pretty big, maybe we should dynamically import it if someone uses mermaid syntax, rather than everyone paying that cost.

Reply:
Should be fixed now @pngwn